test/extended/builds/service.go

```diff
@@ -0,0 +1,72 @@
+package builds
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Builds][Conformance] build can reference a cluster service", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc             = exutil.NewCLI("build-service", exutil.KubeConfigPath())
+		testDockerfile = `
```

Comment:
Just a nit, but I like to use heredoc for this so that the indenting is easier to read.